### ISSUE TITLE
cmd+v to paste not working

### ISSUE THREAD

**Issue by haifengkao** [rhysd/neovim-component#32]
Expected
----------
From neovim-component 0.6.2
![2016-08-10 12 36 48](https://cloud.githubusercontent.com/assets/4080524/17541960/2135b9e8-5ef7-11e6-8dc6-ded9b64d7eae.jpg)

Actual
------
From neovim-component 0.7.3
![2016-08-10 12 39 00](https://cloud.githubusercontent.com/assets/4080524/17541996/6f7e70fe-5ef7-11e6-91ce-52df90af7112.jpg)

Step
-----
- use visual mode to copy the text (cmd+c not working as well)
- paste the content by cmd+v

**Comment by rhysd** [comment]
Yes, because Cmd+v is now sent to `nvim` as `<D-v>` sequence.  If you don't like the behavior, please add `disable-meta-key` attribute to `<neovim-editor>` in your HTML.  It stops sending `<D-...>` sequence and app can handle Cmd key.

**Comment by haifengkao** [comment]
I have added the following
``` html
    <neovim-editor id="nyaovim-editor" argv="[[argv]]" font-size="16" font="Powerline Consolas" blink-cursor="false" disable-meta-key="true"></neovim-editor>
```
But it didn't work :(

**Comment by rhysd** [comment]
`disable-meta-key` is a boolean attribute.  So please remove `="true"` as below:

```html
<neovim-editor ... no-blink-cursor disable-meta-key></neovim-editor>
```

`blink-cursor` had the same issue.  So I renamed it to `no-blink-cursor`.  Please try it also.

**Comment by haifengkao** [comment]
still not working
```
    <neovim-editor id="nyaovim-editor" argv="[[argv]]" font-size="16" disable-meta-key font="Powerline Consolas"></neovim-editor>

```

**Comment by rhysd** [comment]
I guess this was fixed by the fix for https://github.com/rhysd/NyaoVim/issues/94.
